config/packager.php

```diff
@@ -8,6 +8,8 @@
      */
     'skeleton' => 'http://github.com/Jeroen-G/packager-skeleton/archive/master.zip',
 
+    'curl_verify_cert' => env('CURL_VERIFY', true),
```

Comment:
Could you place this above this as a nice looking comment block?

> When you run into issues downloading the skeleton, this might be because of a file regarding SSL certificates missing on the (Windows) OS. This can be solved by setting the verification of the certificate to false. Of course this means it will be less secure.